### DISCUSSION TITLE
如何给RichText渲染的富文本内容添加点击监听事件

### DISCUSSION THREAD

**Discussion by Lynne42** [NervJS/taro#16163] [Q & A]
<RichText  nodes={"抱歉！<span data-content='热热热'>没有能够帮到您</span>"} /> 比如这是要显示的富文本，里面有一些标签， 在点击这些标签的时候触发点击事件
如何能够添加点击事件监听

**Comment by honlyHuang** [comment]
渲染 HTML https://docs.taro.zone/docs/html
